src/ensembl/src/content/app/contact-us/ContactUs.scss

```diff
@@ -0,0 +1,43 @@
+.wrapper {
```

Comment:
style: why 4-space indent?

Reply:
Fixed it. VScode says 2 spaces. Not sure how it ended up with 4 spaces.